### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: src/Elmish.WPF/Binding.fs [hunk 1]
```diff
@@ -73,34 +73,34 @@ module internal BindingLogic =
 
             update curTarget nextSource (curTargetIdx + 1)
             update nextTarget curSource curTargetIdx
-
```
Comment (Evangelink) [inline src/Elmish.WPF/Binding.fs:76]: Crap sorry, I forgot to disable "trailing whitespace visualizer", let me revert the unwanted changes.

**Comment by cmeeren** [comment]
Thanks! (For the record, "centred" is valid English)

**Comment by Evangelink** [comment]
@cmeeren Thanks! I saw you were using more of the British English than the American English but I never saw `centred` so sorry about this one.

**Comment by cmeeren** [comment]
No worries, I'm not a native English speaker and don't really have any strong opinions on US vs. UK spellings. I try to be somewhat consistent, but I find myself fluctuating a bit between the two styles in different periods of my life.

**Pull request merged**